src/components/Modals/Subjects/Subjects.js

```diff
@@ -42,68 +43,80 @@ export default function Subjects ({ subjects = mockSubjects }) {
         <Text color='kelp'>Associated subjects ({subjects.length})</Text>
         <PlainButton
           icon={<Close color='black' size='small' />}
-          onClick={() => console.log('Close the modal')}
+          onClick={() => onClose()}
           text='Close'
           reverse
         />
       </Box>
-      <Neuton>Click a subject to view it on Zooniverse Talk</Neuton>
+      <Neuton>Click a subject to view more information</Neuton>
       <Box
         direction='row'
         justify='between'
         width='medium'
         wrap
       >
-        {chunkedSubjects[subjectIndex].map((subject, i) => {
+        {currentPage.map((subject, i) => {
           return (
             <Box
               key={`SUBJECTS_${i}`}
               basis='30%'
               margin={{ bottom: 'xsmall' }}
               height='5.5em'
             >
-              <Anchor href={subject.link}>
+              <Button
+                onClick={() => {
+                  onClose()
+                  onSelectSubject(subject)
+                }}
+                plain
+              >
                 <Image
                   alt={subject.alt}
                   fit='contain'
-                  src={subject.src}
+                  src={`//${subject.subjectMediaLocation}`}
                   width='100%'
                 />
-              </Anchor>
+              </Button>
             </Box>
           )
         })}
       </Box>
-      <Box
-        direction='row'
-        gap='xxsmall'
-        margin={{ horizontal: 'auto', top: 'auto' }}
-      >
-        <Button
-          disabled={subjectIndex === 0}
-          label={<StyledText size='0.5em'>&#9664;</StyledText>}
-          onClick={() => changeSubjectIndex(subjectIndex - 1)}
-          plain
-        />
-        {chunkedSubjects.map((subj, i) => {
-          const char = i === subjectIndex ? `\u25CF` : `\u25CB`
-          return (
-            <Button key={`SUBJECTS_${i}`} onClick={() => changeSubjectIndex(i)}>
-             {char}
-            </Button>
-          )
-        })}
-        <Button
-          disabled={subjectIndex === lastIndex}
-          label={<StyledText size='0.5em'>&#9654;</StyledText>}
-          onClick={() => changeSubjectIndex(subjectIndex + 1)}
-          plain
-        />
-      </Box>
+      {subjects.length > 9 && (
+        <Box
+          direction='row'
+          gap='xxsmall'
+          margin={{ horizontal: 'auto', top: 'auto' }}
+        >
+          <Button
+            disabled={subjectIndex === 0}
+            label={<StyledText size='0.5em'>&#9664;</StyledText>}
+            onClick={() => changeSubjectIndex(subjectIndex - 1)}
+            plain
+          />
```

Comment:
I'd add an aria label here via  `a11yTitle`  prop to indicate what this does even  though there's a defined label. I'm not sure if screen readers read out unicode and even if they do, "left arrow" doesn't really say what this is for.